chia/consensus/block_body_validation.py

```diff
@@ -478,7 +479,9 @@ async def validate_block_body(
             return Err.BAD_AGGREGATE_SIGNATURE, None
 
         # noinspection PyTypeChecker
-        if not AugSchemeMPL.aggregate_verify(pairs_pks, pairs_msgs, block.transactions_info.aggregated_signature):
+        if not cached_bls.aggregate_verify(
+            pairs_pks, pairs_msgs, block.transactions_info.aggregated_signature, isinstance(block, UnfinishedBlock)
```

Comment:
maybe use a variable and add a comment here:
```python
# We force for unfinished block, since adding them to the cache is useful. For finished blocks, we might be syncing and therefore cache might not be useful.
force: bool = isinstance(block, UnfinishedBlock)
```

Reply:
Fixed